volontulo/templates/auth/password_reset.html

```diff
@@ -0,0 +1,16 @@
+{% extends "common/col1.html" %}
+
+{% block title %}Resetowanie hasła{% endblock %}
+
+{% block content %}
+    <h2>Resetowanie hasła</h2>
+
+    <form id="user_register" method="post" action="{% url 'password_reset' %}" role="form" enctype="multipart/form-data">
+        {% csrf_token %}
+        <div class="form-group">
+            {{ user_form.email.label_tag }}
+            <input type="text" value="" name="email" class="form-control" id="{{ user_form.email.id_for_label }}" />
```

Comment:
Should it work like this already? It there will not be request.POST.email template engine will substitute here empty string.